packages/react/.eslintrc.js

```diff
@@ -10,6 +10,7 @@ module.exports = {
 
     // NOTE: remove to turn on linting for test files
     '__tests__',
+    'scripts',
```

Comment:
Why do we need to ignore the _scripts_ file?

Reply:
because the scripts aren't included in the tsconfig file and ESLint really doesn't like that! I didn't think it made sense to start compiling the scripts file but I'm open to suggestions.

Reply:
Full error:
```bash
/Users/reesscot/amplify-ui/packages/react/scripts/generatePrimitiveCatalog.ts
  0:0  error  Parsing error: ESLint was configured to run on `<tsconfigRootDir>/scripts/generatePrimitiveCatalog.ts` using `parserOptions.project`: <tsconfigRootDir>/../../../../../users/reesscot/amplify-ui/packages/react/tsconfig.json
However, that TSConfig does not include this file. Either:
- Change ESLint's list of included files to not include this file
- Change that TSConfig to include this file
- Create a new TSConfig that includes this file and include it in your parserOptions.project
See the TypeScript ESLint docs for more info: https://typescript-eslint.io/docs/linting/troubleshooting##i-get-errors-telling-me-eslint-was-configured-to-run--however-that-tsconfig-does-not--none-of-those-tsconfigs-include-this-file
```
Can we follow up on this if we want to include it? The ESLint error only failed because I touched the file.